lib/tasks/factory_girl.rake

```diff
@@ -1,15 +0,0 @@
-namespace :factory_girl do
-  desc "Verify that all FactoryGirl factories are valid"
-  task lint: :environment do
-    if Rails.env.test?
-      begin
-        DatabaseCleaner.start
-        FactoryGirl.lint
-      ensure
-        DatabaseCleaner.clean
-      end
-    else
-      system("bundle exec rake factory_girl:lint RAILS_ENV='test'")
-    end
-  end
-end
```

Comment:
Ah no worries. To be clear all you needed to do was update the root `Rakefile` here: https://github.com/spartansystems/policy-mirror/blob/master/Rakefile#L12 to not have `factory_girl:lint`